packages/aws-library/src/aws_library/ec2/client.py

```diff
@@ -274,6 +275,24 @@ async def get_instances(
         )
         return all_instances
 
+    async def stop_instances(self, instance_datas: Iterable[EC2InstanceData]) -> None:
+        try:
+            with log_context(
+                _logger,
+                logging.INFO,
+                msg=f"stopping instances {[i.id for i in instance_datas]}",
+            ):
+                await self.client.stop_instances(
+                    InstanceIds=[i.id for i in instance_datas]
+                )
+        except botocore.exceptions.ClientError as exc:
+            if (
+                exc.response.get("Error", {}).get("Code", "")
+                == "InvalidInstanceID.NotFound"
```

Comment:
THOUGHT: I wonder if you could also keep these error codes in _errors ?

Reply:
I did not refactor the EC2 client. maybe in a next PR to have them on par.